tests/bsim/bluetooth/host/iso/bis/src/common.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <string.h>
+
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/addr.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/sys/util.h>
+
+#include "babblekit/flags.h"
+#include "babblekit/testcase.h"
+
+#include "common.h"
+
+extern enum bst_result_t bst_result;
+
+void test_init(void)
+{
+	bst_result = In_progress;
+	bst_ticker_set_next_tick_absolute(WAIT_TIME);
+}
+
+void test_tick(bs_time_t HW_device_time)
```

Comment:
I think you could use the `test_delete_f` callback instead (see https://github.com/jori-nordic/zephyr/blob/f2e3d3f951112ffec7f3a1a40bcf0c187dcae40e/tests/bsim/bluetooth/host/misc/sample_test/src/main.c#L18-L35).
Then you don't need `common.h` at all (since you won't need `test_init()` any more)

Reply:
Hmm, looked into this. 

I would then either have to implement this function in each source file or still need the common files, so it doesn't really change anything in terms of files needed or structure. 

There is also the alternative that some tests are using where they keep the common functionality in `main.c` as the above and replace the header file with `extern` instead. Not sure if that is really better.

Reply:
yeah it was just a nit, don't need to act on it.
For next time maybe :+1: